_posts/2016-08-26-babili.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title:  "Releasing Babili (beta)"
+author: Henry Zhu
+date:   2016-08-26 09:30:00
+categories: announcements
+share_text: "Releasing Babili (beta)"
+---
+
+Babili (babel-minify)
+
+We released [babili](https://github.com/babel/babili) yesterday! (under an MIT license)
+
+There seem to be a lot of (valid) questions about why a new minifier is necessary so we want to write a post about some reasons and the future.
+
+## Why minify?
+
+## Issues with current minifiers
+
+Tools such as [uglify](https://github.com/mishoo/UglifyJS2) don't currently support targeting the latest version of Javascript ([yet](https://github.com/mishoo/UglifyJS2/issues/448)).
+
+We currently to use tools like Babel to compile ES6 code down to ES5 code to support older browsers. Then we use something like uglify to minify our code to cut down on the bundle size.
+
+However as browsers implement more ES6 features and we drop support for older browser versions, there will be a point where you wouldn't have to compile ES6 code to ES5 because all the support targets already understand ES6. However b ecause uglify cannot parse ES6, you would still have to compile down to ES5 anyway.
```

Comment:
😱 I had no idea! 

I think that saying "there will be a point where you wouldn't have to compile ES6 code to ES5" should maybe be left out for two reasons:
1. It's a little presumptuous since the community seems content to use transpilers to get the latest and greatest
2. Since babel6 babel can use plugins to do all kinds of things and it will be good to keep it in the toolchain.

Re: the uglify bit - that needs to be an h1 at the top of the page in big bold block letters. I had no clue and that would have been pretty annoying to find out on my own.